src/System.Drawing.Common/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.1.1</AssemblyVersion>
```

Comment:
do we need to bump the assmembly version also ?

Reply:
Yes. I'll update the PR.

Reply:
Fixed.